lib/sda/protocol.go

```diff
@@ -24,17 +25,13 @@ type ProtocolInstance interface {
 	// Start on it.
 	Start() error
 	// Dispatch is called at the beginning by SDA for listening on the channels
-	// XXX Should remove that => not using ncessarily channels
```

Comment:
That's exactly why this comment is there: to discuss about that. From my point of view, it's not clear at all we should keep that function. 
We can fill an issue about it and discuss about it there ? But we should definitely stick that comment here, since that's not the goal of this PR to decide about that.